frontend/public/components/node.tsx

```diff
@@ -245,9 +244,7 @@ const pages = [
   pods(({obj}) => <PodsPage showTitle={false} fieldSelector={`spec.nodeName=${obj.metadata.name}`} />),
   events(ResourceEventStream),
 ];
-export const NodesDetailsPage = props => <DetailsPage
+export const NodesDetailsPage = props => <NodeDetailsPageWithMaintenance
   {...props}
```

Comment:
why not pass menuActions as a prop? Then they would not have to be exported.

Reply:
no reason, just forgotten :) `menuActions` are now passed via `props`